src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/ExportToolbarAction.js

```diff
@@ -49,7 +50,9 @@ export default class ExportToolbarAction extends AbstractToolbarAction {
                             >
                                 <SingleSelect onChange={this.handleEnclosureChanged} value={this.enclosure}>
                                     <SingleSelect.Option value={'"'}>&quot;</SingleSelect.Option>
-                                    <SingleSelect.Option value=" ">&nbsp;</SingleSelect.Option>
+                                    <SingleSelect.Option value="space">
+                                        {translate('sulu_admin.enclosure_space')}
```

Comment:
Is that really a space? I thought if you choose this option it doesn't add any character, not even a space?

Reply:
Yes you are right. I misinterpreted it because the attribute was `value=" "` so it wraps the words inside two spaces. If we don't want any character at all we have to use `value=""`.